### ISSUE TITLE
fcn16s model mean IU lower than fcn32s model

### ISSUE THREAD

**Issue by Hjy20255** [wkentaro/fcn#89]
I'm training FCN16s using pretraind FCN-32s with meanIU ~57 %. The meanIU of FCN16s is 47 % after 100000 iterations. It is lower than meanIU of FCN-32s.
I'm using the following parameters.
・learning rate: 1e-10
・momentum: 0.99
・weight decay: 5e-4
also i notice  #79  ,so i add some code. I  copy the weight of pre-trained model (VGG) to FCN32s 
   `for link_name in ['fc6', 'fc7']:
        W1, b1 = getattr(vgg, link_name).params()
        W2, b2 = getattr(model, link_name).params()
        W2.data = W1.data.reshape(W2.shape)
        b2.data = b1.data` 
But FCN-16s is lower than meanIU of FCN-32s.

**Comment by Hjy20255** [comment]
fcn-16s
![_016](https://cloud.githubusercontent.com/assets/17898101/24129908/9daa00dc-0e20-11e7-83a9-8ec2459f8416.png)

**Comment by Hjy20255** [comment]
fcn-32s
![_017](https://cloud.githubusercontent.com/assets/17898101/24130020/343182dc-0e21-11e7-822f-b5977634b9d7.png)

**Comment by wkentaro** [comment]
It works if you use latest master branch:
![screen shot 2017-06-07 at 4 05 17](https://user-images.githubusercontent.com/4310419/26847021-8aa6a406-4b36-11e7-8c91-461640d59312.png)
